### PR TITLE
Issue 69/add flowing list

### PR DESCRIPTION
**Pull request by mmircea16**
The existent List component assumes that the number of columns is fixed. In order to render well in the lower resolution screens we might need the ability to have a fluid list that goes on the next line when it fills a row.

Closes https://github.com/code4romania/taskforce-fe-components/issues/69

Please mention the main changes this PR brings:

- adds FlowingList component for items that need to easily flow on the next row(s) when there is not enough space to render them on one row

### CODE REVIEW

File: src/components/flowing-list/flowing-list.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./flowing-list.styles.scss";
+
+export const FlowingList = ({ children }) => {
+  return <div className={"__flowing-list"}>{children}</div>;
```
Comment (surdu) [inline src/components/flowing-list/flowing-list.js]: I keep seeing this. Is there a difference between `attribute={"value"}` and `attribute="value"` ?

Comment (mmircea16) [reply src/components/flowing-list/flowing-list.js]: No difference, it's there as this seems to be style used. I don't mind either way

**Comment by oanamun** [comment]
Shouldn't we update the existing List component instead of creating a new component? Also I checked 'Stam acasa' and I can't reproduce this problem. Can you check?

**Comment by mmircea16** [comment]
It was fixed in the yesterday's bug smash so this is obsolete.

I will close this pull request.

**Pull request closed without merging**